client/java/src/main/java/io/openlineage/client/metrics/SimpleMetricsBuilder.java

```diff
@@ -0,0 +1,34 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.metrics;
+
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import java.util.Map;
+
+/**
+ * This class implements the MetricsBuilder interface with SimpleMeterRegistry as its type.
+ * SimpleMeterRegistry is a type of MeterRegistry, designed for testing functionality that does not
+ * require a backend monitoring system.
+ */
+public class SimpleMetricsBuilder implements MetricsBuilder<SimpleMeterRegistry> {
```

Comment:
SimpleMeterBuilder ?

Reply:
Taken this offline with Paweł and renamed the classes accordingly.